dd-trace-propagation/benches/inject_benchmark.rs

```diff
@@ -97,7 +109,7 @@ fn bench_datadog_only_inject<M: criterion::measurement::Measurement + Measuremen
         b.iter_batched(
             || (create_simple_span_context(), BenchInjector::new()),
             |(mut context, mut carrier)| {
-                propagator.inject(black_box(&mut context), black_box(&mut carrier))
+                propagator.inject(&mut span_context_to_inject(&mut context), &mut carrier)
```

Comment:
Does removing these `black_box` calls change the performance of the benchmark?

Reply:
The black box did nothing because criterion already wraps inputs and outputs of the measured function in black_box

Reply:
Interesting. Didn't know that.